### ISSUE TITLE
"Teleporting" to a new domain or manually entering URL does not remove current environment

### ISSUE THREAD

**Issue by Aitolda** [vircadia/vircadia-web#199]
If you try to manually change the URL both your current environment and the one you teleport two will layer on top of each other.

For example,  If you are at app.vircadia.com/#/antares.digisomni.com and swap the url for app.vircadia.com/#/nautatuma.vircadia.com the content of antares will persist when the content of nautatuma loads. Creating an inworld teleport yields the same results.

**Comment by digisomni** [comment]
Ah so the rabbit hole goes deeper. It would seem that between disconnects that the entities are not cleared fully. Does this still happen with master?

**Comment by Aitolda** [comment]
> Ah so the rabbit hole goes deeper. It would seem that between disconnects that the entities are not cleared fully. Does this still happen with master?

Yes.  However, it might actually somehow be reloading the world again before it moves to the next.  It looks very much like when you first go to a domain and it loads, then kicks you back to the loading screen and loads again.   That is to say when you try to go to another environment, it seems like it does the same thing BEFORE you leave.  Appears to reload just before it moves to the next server.   Granted appearance can be deceptive.
